### PR TITLE
benchmark

### PR DESCRIPTION
**Pull request by 3nids**

### CODE REVIEW

File: docker-compose.arm64.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+
+services:
+  postgres:
+    image: duvel/postgis:12-2.5-arm64
```
Comment (suricactus) [inline docker-compose.arm64.yml:5]: Any reason to prefer such an ancient version?

File: scripts/restart.sh [hunk 2]
```diff
@@ -1,19 +1,22 @@
...
 SIZE=${1:-100}
 
-rm src/tests/migrations/0*.py || true
 
 docker compose down --volumes || true
 
 docker compose up --build --force-recreate -d
 sleep 5
 
+rm src/tests/migrations/0*.py || true
```
Comment (suricactus) [inline scripts/restart.sh:14]: Why is that? What if you have more than 100 migrations?

File: src/tests/models.py [hunk 3]
```diff
@@ -11,10 +10,11 @@
 logger = logging.getLogger(__name__)
 
 
-@register_oapif_viewset(crs=2056)
-class Point_2056_10fields(ComputedFieldsModel):
+class BaseModel10Fields(models.Model):
```
Comment (suricactus) [inline src/tests/models.py:13]:
```suggestion
class BaseModelWithTenFields(models.Model):
```

Slighthy more verbose, but more explicit. I initialy thought it's something like `i18n` or `l10n`

**Review by suricactus — COMMENTED** [review]
Looks good. Did not check the graph generation code at all, but I believe you know all the details to have constructed it properly.

Small comments here and there more about styling than actual problems.

Nice to have official benchmarking. The optimization seems considerable!

File: docker-compose.arm64.yml (hunk 1 above)
Comment (3nids) [reply docker-compose.arm64.yml:5]: ARM images are not available yet, but here is some fresh update
https://github.com/postgis/docker-postgis/issues/216#issuecomment-1763399631

File: scripts/download-fixtures.sh [hunk 4]
```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -e
+
+mkdir -p src/tests/fixtures
+
+# download fixtures
+curl -L -o ./src/tests/fixtures/polygon_2056.json.gz 'https://drive.google.com/uc?export=download&id=1UuGoK_9Y99jiTvQd4juxu85eVZhvlAvy'
+curl -L -o ./src/tests/fixtures/polygon_2056_local_geom.json.gz 'https://drive.google.com/uc?export=download&id=18PGtiptcJiRtLnVq7N64EVQLagse0bu0'
```
[the remark this answers is not in the document]
Comment (3nids) [reply scripts/download-fixtures.sh:9]: The file is 20mb zipped.

Dataset comes from here: https://datahub.io/cividi/ch-municipalities#resource-gemeinden-geojson
If we reuse django-geojson (see #104 ) that could be an easy way to reuse the original file

**Pull request merged**